src/Data/Dwarf.hs

```diff
@@ -165,7 +149,10 @@ data CUContext = CUContext
     cuSections :: !Sections,
     cuDieOffset :: !DIEOffset,
     cuNextOffset :: !CUOffset,
-    cuNextAbbrevCache :: !(M.Map Word64 (M.Map AbbrevId DW_ABBREV))
+    cuNextAbbrevCache :: !(M.Map Word64 (M.Map AbbrevId DW_ABBREV)),
+    cuUnitType :: !(Maybe Word8),
+    cuAddrBase :: !(Maybe Word64),
+    cuStringOffsetBase :: !(Maybe Word64)
```

Comment:
Could you add Haddock comments on each of these fields describing what they represent, as well as why they use `Maybe`?

(This isn't really your fault, since most of the other fields of `CUContext` aren't commented either, but I think it would be worth doing this for any new definitions that we add going forward.)